enhancements/sig-policy/71-hub-templates-auto-reconcile/README.md

```diff
@@ -0,0 +1,232 @@
+# Automatic Reconciliation When Hub Template Referred Object is Updated
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in
+      [website](https://github.com/open-cluster-management-io/open-cluster-management-io.github.io/)
+
+## Summary
+
+Policies support dynamic values through policy templating. One such option is Hub policy templating, which are templates
+that are executed on the Hub before the policy is distributed to the managed clusters. Hub policy templating supports
+Kubernetes API queries such as getting a field from a ConfigMap.
+
+The current limitation is that if a referenced object in the Hub policy template is updated, it does not immediately
+cause the policy to be reconciled with this update. This does happen every ten hours or every time the policy is
+reconciled such as when the policy status is updated or the policy applies to a new cluster. A workaround exists where
+the “policy.open-cluster-management.io/trigger-update” annotation can be updated on the policy to cause the policy to be
+reconciled immediately.
+
+The current options that trigger the policy to be reconciled are insufficient and it should happen automatically and
+immediately when the referenced object is updated.
+
+## Motivation
+
+A common use-case for Hub policy templates is to copy a Secret from the Hub to a managed cluster. If the Secret on the
+Hub is updated, it is desired for the Secret to be immediately updated on the managed cluster. This isn’t possible
+without the `policy.open-cluster-management.io/trigger-update` annotation being manually updated or waiting for up to
+[ten hours](https://github.com/kubernetes-sigs/controller-runtime/blob/02dc46444cfe0e8bbe61bb3d4c3a12a0faadcd24/pkg/manager/manager.go#L109-L134).
+
+### Goals
+
+1. A policy is automatically reconciled when it has a Hub policy template that references an object that was updated.
```

Comment:
For managed side template policy,  is it supported ?

Reply:
It wouldn't be since on the managed side, policies are usually evaluated every 10-20 seconds, so the extra overhead isn't worth it in my opinion.

If a user did have a longer evaluation interval on their policy than the default, then I think it would be a nice feature, but I'd rather defer this for now.

Reply:
I think this should be doc'd in the 2.7 doc.